react-app/src/components/HomeFeed/HomeFeed.js

```diff
@@ -22,15 +22,15 @@ const HomeFeed = () => {
     await dispatch(getAllWines())
       .then(() => setAllWines(Object.values(wines).reverse()))
       .then(() => dispatch(getAllReviews()))
-      .then(() => setNewReviews(reviewSort(Object.entries(reviews).reverse())))
+      .then((ret) => setNewReviews(reviewSort(Object.entries(ret).reverse())))
       .then(() => setLoaded(true))
   }, [dispatch, wines, reviews])
 
   useEffect(() => {
     dispatch(getAllWines())
       .then(() => setAllWines(Object.values(wines).reverse()))
       .then(() => dispatch(getAllReviews()))
-      .then(() => setNewReviews(reviewSort(Object.entries(reviews).reverse())))
+      .then((ret) => setNewReviews(reviewSort(Object.entries(ret).reverse())))
```

Comment:
I think this might be the only line here that actually needs to be a .then since most of the rest of this is setting state which isn't asynchronous lol. but this works and I'm not gonna mess with it